package.json

```diff
@@ -38,8 +38,8 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-footer": "10.2.1",
     "@edx/frontend-component-header": "^3.3.0",
-    "@edx/frontend-platform": "1.15.2",
-    "@edx/paragon": "19.25.3",
+    "@edx/frontend-platform": "^3.0.1",
+    "@edx/paragon": "^20.17.0",
```

Comment:
[20.0.0](https://github.com/openedx/paragon/releases/tag/v20.0.0): The same general concern about react-intl@v5 applies, in particular with regards to testing.  It looks like we're already doing the right thing with [ctxMount](https://github.com/openedx/frontend-app-library-authoring/blob/master/src/library-authoring/common/specs/helpers.jsx#L153) and `ctxRender`, but it's worth double-checking.